app/views/steps/income/income_payments/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+<% title t('.page_title') %>
+<% step_header(path: crime_applications_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <span class="govuk-caption-xl"><%= t('steps.income.caption') %></span>
+    <%= govuk_error_summary(@form_object) %>
+
+    <%= step_form @form_object do |f| %>
+      <%= f.govuk_check_boxes_fieldset :income_payments, legend: { tag: 'h1', size: 'xl' } do |x| %>
+
+        <% @form_object.ordered_payment_types.each do |type| %>
+          <%= f.govuk_check_box :types, type.to_s, checked: f.object.checked?(type), link_errors: true do %>
+            <%= f.fields_for type.to_sym, f.object.public_send(type.to_s) do |g| %>
+              <%= g.govuk_fieldset legend: nil, id: "income_payment_#{type}" do %>
```

Comment:
Does the legend have to be nil or can it just not be set?

Reply:
Without the fieldset the form field naming scheme changes and therefore the form breaks

Reply:
Should the legend actually be nil or just be hidden? Just wondering if this may cause problems with accessibility. It's quite a complex form so wondering if it still makes sense without any legend or if it would be better to have `#{payment_type} details` or something like this, just a suggestion though